backend/src/main/java/com/codesquad/todolist/card/CardController.java

```diff
@@ -19,15 +31,40 @@ public CardController(CardService cardService) {
         this.cardService = cardService;
     }
 
+    @ApiOperation(value = "카드 추가", notes = "새로운 카드를 지정한 컬럼에 추가합니다.")
     @PostMapping
-    public void create(CardCreateRequest createRequest) {
-        cardService.create(createRequest);
+    public ResponseEntity<HistoryResponse> createCard(
+        @RequestBody @Valid CardCreateRequest request) {
+        HistoryResponse historyResponse = cardService.create(request);
+        return new ResponseEntity<>(historyResponse, HttpStatus.CREATED);
     }
 
+    @ApiOperation(value = "카드 업데이트", notes = "지정한 카드를 새로운 데이터로 업데이트합니다.")
     @PutMapping("/{id}")
-    public void updateCard(@PathVariable(value = "id") Integer cardId,
-        @RequestBody CardUpdateRequest request) {
-        cardService.update(cardId, request);
+    public HistoryResponse updateCard(
+        @ApiParam(name = "id", value = "업데이트할 카드 Id", required = true)
+        @PathVariable(value = "id") Integer cardId,
+        @RequestBody @Valid CardUpdateRequest request) {
+        return cardService.update(cardId, request);
+    }
+
+    @ApiOperation(value = "카드 이동", notes = "카드를 현재 컬럼 혹은 다른 컬럼으로 이동하고 순서를 변경합니다.")
+    @PutMapping("/{id}/move")
+    public ResponseEntity<?> moveCard(
```

Comment:
p2) 여기도 아래처럼 가는게 좋지 않을까요?

```suggestion
    public ResponseEntity<HistoryResponse> moveCard(
```

Reply:
넵! 수정하겠습니다. 굳이 와일드카드 매개변수화 타입을 사용할 필요가 없었네요.